examples/ct/structures_test.go

```diff
@@ -43,4 +58,54 @@ func TestGetCTLogIDNotLoaded(t *testing.T) {
 }
 
 func TestSignV1SCTForCertificate(t *testing.T) {
+	cert, err := fixchain.CertificateFromPEM(testonly.LeafSignedByFakeIntermediateCertPem)
```

Comment:
OK, since you're happy with the dependency on google/certificate-transparency from files under examples/ct, then this seems fine.  Maybe you can comment somewhere what this dep brings to the party.